src/main/java/com/vinylteam/vinyl/util/CaptchaValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.vinylteam.vinyl.util;
+
+import com.octo.captcha.service.image.DefaultManageableImageCaptchaService;
+import com.octo.captcha.service.image.ImageCaptchaService;
+
+public class CaptchaValidator {
+    private static ImageCaptchaService instance = new DefaultManageableImageCaptchaService();
+
+    public static ImageCaptchaService getInstance() {
+        return instance;
+    }
```

Comment:
Please add empty row after interface/class declaration and before closing. It is our code style